docs/en/developer/00-drivers/03-jdbc.md

```diff
@@ -101,6 +101,35 @@ public class demo {
         System.exit(0);
 ```
 
+### Example: Copy into or merge into table
+
+```java
+    public void copyInto(String tableName, List<String> files) throws Exception {
+        String filesStr = "'" + String.join("','", files) + "'";
+        String copyIntoSql = String.format("copy into %s from @~ files=(%s) file_format=(type=NDJSON) purge=true;", tableName, filesStr);
+        Connection connection = createConnection();
+        try (Statement statement = connection.createStatement()) {
+            Instant copyIntoStart = Instant.now();
+            statement.execute(copyIntoSql);
+            ResultSet r = statement.getResultSet();
+            while (r.next()) {
+            }
+            Instant copyIntoEnd = Instant.now();
+            System.out.println("Copied files into: " + files.size() + " , time elapsed: " + (copyIntoEnd.toEpochMilli() - copyIntoStart.toEpochMilli()) + "ms");
+        } catch (Exception e) {
+            e.printStackTrace();
+        } finally {
+            connection.close();
+        }
+    }
+// For merge into just replace the copyIntoSql.
+```
+:::tip
+1. Because the select, copy inot, merge into are query type SQL, they will return a ResultSet object, you must call rs.next() before accessing the data. Otherwise, the query may be canceled. If you do not want get the result, you can call while(r.next(){}) to iterate over the result set.
```

Comment:
- Because the SQL commands such as SELECT, COPY INTO, and MERGE INTO return a ResultSet object, it is necessary to call rs.next() before accessing the data. Failure to do so may result in the query being canceled. If you don't intend to retrieve the results, you can iterate over the ResultSet using a while loop (while (r.next()){}) to avoid this issue.